server/eclipse-project/src/main/java/us/freeandfair/corla/model/RLAAlgorithm.java

```diff
@@ -144,7 +158,8 @@ public RLAAlgorithm(final CountyDashboard the_dashboard) {
     final Long max_long = 
         CastVoteRecordQueries.countMatching(my_dashboard.cvrUploadTimestamp(), 
                                             my_dashboard.countyID(),
-                                            RecordType.UPLOADED);
+                                            RecordType.UPLOADED) - 1;
```

Comment:
It was wrong in the first place; Joey's code generates a random sequence of the numbers from `min` to `max` inclusive, not from `min` to `max - 1`.